packages/marko-web/middleware/with-content.js

```diff
@@ -13,8 +13,11 @@ module.exports = ({
   const id = isFn(idResolver) ? await idResolver(req, res) : req.params.id;
   const { apollo } = req;
 
-  const additionalInput = {};
-  if (req.cookies['preview-mode'] || req.query['preview-mode']) additionalInput.status = 'any';
+  const additionalInput = { since: new Date() };
+  if (req.cookies['preview-mode'] || req.query['preview-mode']) {
+    delete additionalInput.since;
```

Comment:
Add an else statement and add the date there. Instead of always adding and then deleting.